transit_odp/browse/templates/browse/disruptions/organisation/disruption_detail.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'browse/base.html' %}
+
+{% load static %}
+{% load i18n %}
+{% load breadcrumbs %}
+
+{% block title_tag %}
+  {% trans 'Disruption Detail' %}
+{% endblock %}
+
+{% block breadcrumb.content %}
+{% endblock %}
+
+{% block backlink %}
+  {% url 'org-detail' orgId=org_id host host.name as backlink_url %}
+  {% include 'snippets/backlink.html' with url=backlink_url %}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl dont-break-out">
+    {% trans 'Disruption detail' %}
+  </h1>
+  {% include 'publish/snippets/dataset_map.html' %}
+  <h3 class="govuk-heading-l govuk-!-padding-top-8">{{ object.summary|capfirst }}</h3>
+  {%  if object.disruptionNoEndDateTime == "true" %}
+    <p class="govuk-body">
+      <img class="govuk-!-margin-right-1" width="20" height="20" src="{% static 'frontend/images/calendar-icon.png' %}"
+           alt="{% trans 'Calendar icon' %}">
+      {{ object.publishStartDate }} {{ object.publishStartTime }} onwards
+    </p>
+  {% else %}
+    <p class="govuk-body">
+      <img class="govuk-!-margin-right-1" width="20" height="20" src="{% static 'frontend/images/calendar-icon.png' %}"
+           alt="{% trans 'Calendar icon' %}">
+      {{ object.publishStartDate }} {{ object.publishStartTime }} - {{ object.publishEndDate }} {{ object.publishEndTime }}
+    </p>
+  {% endif %}
+
+  <p class="govuk-body">{{ object.description|capfirst }}</p>
+
+  {% if object.associatedLink %}
+    <div>
+      <table class="govuk-table consumer-property-table">
+        <tbody class="govuk-table__body">
+        <tr class="govuk-table__row">
+          <th scope="row" class="govuk-table__header">{% trans 'Associated link' %}</th>
+          <td class="govuk-table__cell" colspan="2"><a class="govuk-link"
+                                                       href="https://www.flooding.com">{{ object.associatedLink }}</a></td>
```

Comment:
```suggestion
                                                       href={{ object.associatedLink }}>{{ object.associatedLink }}</a></td>
```
I don't think it should be flooding.com